### ISSUE TITLE
Development setup notes: 'installing from git' link not working properly

### ISSUE THREAD

**Issue by harshhx17** [coala/coala#5992]
The link reloads the entire page and doesn't even redirect to the required location
Instead it should just be '#installing-from-git'

**Comment by abhishalya** [comment]
@harshhx17 It works fine for me. If I'm correct, [this link](https://api.coala.io/en/latest/Developers/Development_Setup.html#installing-from-git) redirects to correct part of the webpage.

**Comment by harshhx17** [comment]
@abhishalya 
The link that you have given, works perfectly fine...
Now there are two things that I wanted to say,
1. If we keep the entire link in href, the page reloads. The point I want to make is that it is inefficient for the entire page to reload, it should just move to the required part if the link is a reference to the same page. This can easily be achieved by replacing the entire link with just #installing-from-git.
2. Although the link in the master is the one that you say is, the link that is actually present on this page is different.

**Comment by VertexC** [comment]
Hi, is this still a opened bug? @harshhx17 can you assign it to me?

**Comment by ashutoshjv661** [comment]
Can you assign me this issue ?
#5992
My first open source contribution

**Comment by piyush072** [comment]
Can I be assigned to this bug?

**Comment by harshhx17** [comment]
@jayvdb ^

**Comment by abhishalya** [comment]
@harshhx17 Please update the issue title and description since I think it is not very clear what you were trying to convey.

**Comment by vibhor16** [comment]
In which repo can I find the code for the page - https://api.coala.io/en/latest/Developers/Development_Setup.html ? 
I can find the link in the Development_Setup.rst file in coala repo, but don't know if this is the link that is used in the url mentioned above.

**Comment by PrajwalM2212** [comment]
@vibhor16 Have you tried looking at https://github.com/coala/documentation

**Comment by vibhor16** [comment]
@PrajwalM2212  Yes I tried looking into the documentation repo, still I couldn't connect if the content in Development_Setup.rst file is used there.

**Comment by vibhor16** [comment]
This issue can be seen on the same page in the following location as well :
- Under Using venv heading:  Now you can activate the environment and start **the next part**.

**Comment by vibhor16** [comment]
@harshhx17 Could you please assign me this bug?

**Comment by roshnet** [comment]
This issue still persists. Can I be assigned this one please?

**Comment by heshamelsawaf** [comment]
> This issue still persists. Can I be assigned this one please?

I have been trying for 6 days now to get someone to assign it to me as my first issue, but sadly no one ever replies :)

**Comment by bkhanale** [comment]
@heshamelsawaf @roshnet You need to ping `jayvdb` or `li-boxuan` on gitter to assign you the issue.

**Comment by heshamelsawaf** [comment]
> @heshamelsawaf @roshnet You need to ping `jayvdb` or `li-boxuan` on gitter to assign you the issue.

That is exactly what I did, 2 days ago and yesterday(I pinged `li-boxuan` in both). And as far as I know, 'jayvdb' is not available nowadays.

**Comment by abhishalya** [comment]
@heshamelsawaf You should wait then, almost every maintainer is busy nowadays.

**Comment by heshamelsawaf** [comment]
Yeah I guess. I'm just trying to resolve some issues for my GSoC proposal. but anyway I understand they are busy.

**Comment by KVGarg** [comment]
> 
> 
> The link reloads the entire page and doesn't even redirect to the required location
> Instead it should just be '#installing-from-git'

It will not matter even if the link contains the webpage url! The actual issue is that **documentation isn't updated**! On inspecting the develpement_setup.html in browser the actual link which is present there is `https://api.coala.io/en/latest/Developers/Development_Setup.html#id1` instead of `https://api.coala.io/en/latest/Developers/Development_Setup.html#installing-from-git`. 
@abhishalya @harshhx17 Can you guys verify this! 
Inspect the last bullet point of [venv section](https://api.coala.io/en/latest/Developers/Development_Setup.html#using-virtualenv)

**Comment by tyajush** [comment]
I am a new commer please assing me this issue. 
I have understood the issue. I  will make it such that, it just move to the required part if the link is a reference to the same page **by replacing the entire link with just #installing-from-git.**

**Comment by tichnas** [comment]
@harshhx17 @abhishalya Hi, I'm new to coala and would love to work on this issue. Can I please be assigned?
Thanks

**Comment by kartikeysaran** [comment]
Hey is issue still open ??

**Comment by bkhanale** [comment]
> Hey is issue still open ??
> 
> 

Yep, you can work on this if you like :)

**Comment by kartikeysaran** [comment]
> Yep, you can work on this if you like :)

Okay, i have started working on this

**Comment by ghost** [comment]
FYI in the bugs table parent to this thread the assignee for this bug isn't displayed as having been assigned.
